src/odb/src/db/dbGuide.h

```diff
@@ -63,6 +63,7 @@ class _dbGuide : public _dbObject
   dbId<_dbTechLayer> layer_;
   dbId<_dbTechLayer> via_layer_;
   dbId<_dbGuide> guide_next_;
+  bool is_jumper_;
```

Comment:
You need to use the code generator to add new attributes to this class (and for dbNet too). Look at https://github.com/The-OpenROAD-Project/OpenROAD/pull/6076/files#diff-fec1fa330d9ecda4724495361c752616685b78b640a9b631a85b77cda9ae7b78 for a reference.

Reply:
dbNet predates the code generator and is hand written.